plugins/tracker-resources/src/components/issues/edit/CreateSubIssue.svelte

```diff
@@ -123,6 +133,22 @@
           tag: label.tag
         })
       }
+
+      const notification: Notification = {
```

Comment:
Let's move this to utility method, so it will be possible to reuse for any kind of new document created.

Reply:
Done